scripts/lib/mkcloud-libvirt.sh

```diff
@@ -112,8 +112,10 @@ done
 iptables_unique_rule PREROUTING -t nat -p tcp --dport 6080 \\
     -j DNAT --to-destination $net_public.2
 
-iptables_unique_rule FORWARD -d $net_admin.0/24 -j ACCEPT
-iptables_unique_rule FORWARD -d $net_public.0/24 -j ACCEPT
+for x in D I ; do
```

Comment:
OK, thanks for the improved commit message.  It's still not 100% clear from the message why the second run breaks - I guess it's because the libvirt network gets deleted recreated before these rules, is that right? But what about the ordering on boot, where `boot.mkcloud*` are run as part of bootup, or on startup of an existing cloud - can we be sure that it will work right in those cases too? I can't see any obvious problem but I'm a little hazy on the details so it would be good to hear you're confident.

Also, given that this chunk of code inserts the rule in a different manner to the above, not using `iptables_unique_rule`, there should be a comment in the code explaining this, because it's not intuitive.